scripts/rancher/cluster-actions.sh

```diff
@@ -3,125 +3,84 @@
 
 #######################################
 # List clusters managed by Rancher
-# Arguments:
-#   Rancher URL
-#   token
-# Examples:
-#   rancher_list_clusters rancher.random_string.geek xxxxx
 #######################################
 rancher_list_clusters() {
-  local rancherUrl=$1
-  local token=$2
-
   echo "Listing clusters registered in Rancher..."
-  curl -s -k "$rancherUrl/v3/clusters" -H "Authorization: Bearer $token" | jq .
+  kubectl get clusters.provisioning.cattle.io --all-namespaces | awk 'NR>1 {print $2}'
 }
 
 #######################################
 # Create downstream custom cluster in Rancher
 # Globals:
 #   CLUSTER_ID
 # Arguments:
-#   Rancher URL
-#   token
 #   name
 #   version (Kubernetes)
 # Examples:
-#   rancher_create_customcluster rancher.random_string.geek xxxxx demo 'v1.27.16+rke2r1'
+#   rancher_create_customcluster demo 'v1.27.16+rke2r1'
 #######################################
 rancher_create_customcluster() {
```

Comment:
Change to be validated manually (as the CI pipeline is not possible for the moment)

Reply:
I duplicated Rancher Playground track and started it.

Got the error:
![image](https://github.com/user-attachments/assets/271ff346-1c1f-4903-a059-409ceba69ce2)

Reply:
![image](https://github.com/user-attachments/assets/587caad7-0bfe-4017-8aa7-d653c769f4ba)
Its alive

Reply:
I also updated the variables in a main script to get the latest k8s and Rancher 

`# defines variables
K3S_VERSION='v1.30'
CERTMANAGER_VERSION='v1.13.0'
LETSENCRYPT_EMAIL_ADDRESS='john.wick@google.com'
RANCHER_REPOSITORY='latest'
RANCHER_VERSION='2.9.1'
RANCHER_DOMAIN="rancher.${HOSTNAME}.${_SANDBOX_ID}.instruqt.io"
RANCHER_REPLICAS='1'
ADMIN_PASSWORD='Sus3R@ncherR0x'
INGRESS_CLASSNAME='traefik'
DOWNSTREAM_CLUSTER_NAME='demo'
RKE2_K8S_VERSION='v1.27.16+rke2r1'`